prisma/migrations/20220121222436_add_versions/migration.sql

```diff
@@ -0,0 +1,8 @@
+-- CreateTable
+CREATE TABLE "versions" (
```

Comment:
Traditionally we have been doing migrations in their own pull request and service / controller changes in separate  ones. You can if you'd like to separate it out, but this is pretty low-risk and won't be used so it's not necessary.

Reply:
Good call, https://github.com/iron-fish/ironfish-api/pull/596. Probably won't get this PR updated today, but it'll get there.